### PR TITLE
[enterprise-4.13] remove NE entry and add RHCOS entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.13

Issue: n/a

Link to docs preview: http://file.rdu.redhat.com/jdohmann/minor-RN-edits/release_notes/ocp-4-13-release-notes.html#ocp-4-13-toolbox-deprecation

QE review:
- [x] QE has approved this change.

Additional information: HAProxy update has been removed from the 4.13 program, toolbox deprecation as been added

### CODE REVIEW

**Comment by jldohmann** [comment]
@mike-nguyen could you PTAL for QE ack (for the RHCOS entry)? this PR adds a toolbox deprecation notice

**Comment by jldohmann** [comment]
0b0e635..c264298 rebase

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -568,6 +564,9 @@ The `ImageContentSourcePolicy` (ICSP) object is now deprecated. You can now use
 For more information on these new objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror_post-install-preparing-for-users[Configuring image registry repository mirroring].
 
 For more information on converting existing ICSP YAML files to IDMS YAML files, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
+[id="ocp-4-13-toolbox-deprecation"]
+==== Toolbox is deprecated in {op-system}
+The toolbox library is deprecated and support will be removed from a future {product-title} release.
```
Comment (stevsmit) [inline release_notes/ocp-4-13-release-notes.adoc:569]: from a future {product-title} release, or in a future {product-title} release? 

There are other instances of this (the RHV note, and the Wildcard note) , but this is my first time actually encountering it.

**Comment by stevsmit** [comment]
@jldohmann just one nit. Feel free to ignore/merge when you're ready.

**Comment by mike-nguyen** [comment]
LGTM

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -569,6 +565,10 @@ For more information on these new objects, see xref:../post_installation_configu
 
 For more information on converting existing ICSP YAML files to IDMS YAML files, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
 
+[id="ocp-4-13-toolbox-deprecation"]
+==== Toolbox is deprecated in {op-system}
+The toolbox library is deprecated and support will be removed from a future {product-title} release.
```
Comment (travier) [inline release_notes/ocp-4-13-release-notes.adoc:570]: Hum, it's not a library, it's a program (a script).

Comment (jldohmann) [reply release_notes/ocp-4-13-release-notes.adoc:570]: fix is in #57902

**Pull request merged**